oci-ignite-advanced/workshops/tenancy/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+    "workshoptitle": "Ignite your Oracle Cloud Infrastructure Services - ADVANCED",
+    "help": "livelabs-help-oci_us@oracle.com",
+	"tutorials": [
+        {
+            "title": "Introduction",
+            "filename": "./../../intro/intro.md"
+        },
+        {
+            "title": "Get Started",
+            "description": "Login to Oracle Cloud",
+            "filename": "https://oracle-livelabs.github.io/common/labs/cloud-login/cloud-login.md"
+        },
+        {
+            "title": "Ignite Basics: Identity and Access Management",
```

Comment:
Labs should start after the Get Started Lab. Please add Lab 1 in the title and update the Lab numbers in the rest of the lab titles in this file.

Reply:
I will move our Ignite Basics section before the Get Started as it is not an actual lab, but a way to revise the knowledge from the first part of the workshop.